src/app/cube/core/cube-core.module.ts

```diff
@@ -10,7 +10,7 @@ import { UsageStatsService } from './usage-stats/usage-stats.service';
 @NgModule({
   imports: [CommonModule],
   exports: [],
-  providers: [UsageStatsService]
+  providers: [MetricService]
```

Comment:
```suggestion
```
This can be deleted because of the way angular works. You'll notice in each service there is a `@Injectable{providedIn: root}` This allows us to use a service where ever in the client. We don't need to provide the service